src/components/Menu/__snapshots__/Menu.test.tsx.snap

```diff
@@ -29,13 +29,11 @@ exports[`Menu Component renders category correctly 1`] = `
           role="menuitem"
           style="padding-left: 0px;"
           tabindex="-1"
-          title="Category Item 1"
         >
           <div
             class="mia-platform-menu-item-icon"
           >
             <svg
-              alt="PiSun"
```

Comment:
doesn't removing this affect accessibility?

Reply:
svg not support this attribute